packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1319,6 +1340,8 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
     return _layer;
   }
 
+  /// Whether this render object requires an update to its compositing state.
+  bool get needsCompositingBitsUpdate => _needsCompositingBitsUpdate;
```

Comment:
The problem is that if you let folks read these bits, they'll write code that's conditional on these bits, which is almost never correct.  I can show you lots of code in Chrome that used to read these sorts of bits and do all manner of bogus things.  :)